include/boost/winapi/critical_section.hpp

```diff
@@ -125,6 +160,18 @@ typedef struct BOOST_MAY_ALIAS _RTL_CRITICAL_SECTION {
     ULONG_PTR_ SpinCount;
 } CRITICAL_SECTION_, *PCRITICAL_SECTION_;
 
+#else
+
+typedef struct BOOST_MAY_ALIAS CRITICAL_SECTION {
```

Comment:
Better use the same name Windows SDK uses for the structure, i.e. `_CRITICAL_SECTION`.

Reply:
Yes, big Windows SDK uses "_RTL_CRITICAL_SECTION" name in its SDK.
Windows CE SDKs uses CRITICAL_SECTION name.

I don't see any harm in using _RTL_CRITICAL_SECTION here, but IMHO it seems inconsistent with your previous strict approach, and may confuse other developers targeting Windows CE.

Of course, I will use _RTL_CRITICAL_SECTION. But if you have a second thought, no problem, just tell.